cassandra-dac/src/main/java/org/sunbird/cassandra/CassandraOperation.java

```diff
@@ -60,6 +60,16 @@
   public void deleteRecord(
       String keyspaceName, String tableName, Map<String, String> compositeKeyMap);
 
+  /**
+   * @desc This method is used to delete one or more records from Cassandra DB corresponding to
+   *     given list of primary keys
+   * @param keyspaceName Keyspace name
+   * @param tableName Table name
+   * @param identifierList identifier list of primary keys
```

Comment:
Rename deleteRecord to deleteRecords.

Update as follows:

@param identifierList List of primary keys of records to be deleted
@return Status of delete records operation